Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -3909,6 +3910,8 @@ def _warn_if_module_is_disabled(e: DemistoException) -> None:
             else:
                 raise
 
+        if arg_to_number(outputs.get("score")) == 0:  # type: ignore[union-attr]
```

Comment:
arg_to_number raises exception if arg is None. Is it ok?

Reply:
it should because `score` doesn't exist, but maybe i should return that the `score` is not available for the user/host specified

ill update it, good catch

Reply:
apparently `arg_to_number` raises exception only when `required=True` and by default its `False`
I updated the implementation to be more clear, take a look